research/src/pages/radio-button.research.mdx

```diff
@@ -0,0 +1,122 @@
+---
+menu: Analysis
+name: Radio Button
+path: /components/radio-button.research
+---
+
+import Image from '../components/image'
+import Anatomy from '../components/anatomy'
+import Concepts from '../components/concepts'
+import ComponentCoverage from '../components/component-coverage'
+
+<ComponentCoverage component="Radio Button" />
+
+## Introduction
+
+_(For a lenghtier discussion, on which this page was based, please refer to the [original radio button issue](https://github.com/WICG/open-ui/issues/251))_
+
+The research showed that different DS’s treat radio buttons in one of two ways, i.e.:
+
+- as the main component, hosted on a “**radio button group**” component;
```

Comment:
I don't understand the difference between these bullets, maybe this can be reworded to be a bit more clear?

Reply:
I reworded it, maybe it's easier to understand now.